### PR TITLE
Admin View WIP

### PR DESCRIPTION
**Pull request by daniel-sudz**

### CODE REVIEW

File: website/.vscode/settings.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-    "eslint.enable": true
-}
```
[the remark this answers is not in the document]
Comment (Zjjc123) [reply website/.vscode/settings.json:3]: I'll do this to master branch

File: website/src/Firebase/setup.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ const firebaseConfig = {
 /** Documentation at https://firebase.google.com/docs/auth/web/passing-state-in-email-actions#passing_statecontinue_url_in_email_actions */
 const actionCodeSettings: firebase.auth.ActionCodeSettings = {
   // The redirect URL
-  url: "https://ehs-service.org",
+  url: "https://localhost",
```
[the remark this answers is not in the document]
Comment (daniel-sudz) [reply website/src/Firebase/setup.ts:19]: @Zjjc123 you wanna update this quickly or let me do it?

[the remark this answers is not in the document]
Comment (Zjjc123) [reply website/src/Firebase/setup.ts]: welp you got it

File: website/src/Components/Table/adminUsersTable.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+const TableHeaderStyle: string =
+  "px-2 py-2 sm:px-3 sm:py-3 md:px-4 md:py-4 lg:px-5 lg:py-5 border-b-2 border-gray-200 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider";
+
+const AdminUsersTable: React.FunctionComponent<{
+  data: profileAndEmail[];
+  handleView: (userID: string) => void;
+}> = ({ data, handleView }) => {
+  const headers = ["Email", "Name", "Graduation Year"];
+  const [filteredData, setFilteredData] = useState<profileAndEmail[]>([]);
+
```
[the remark this answers is not in the document]
Comment (daniel-sudz) [reply website/src/Components/Table/adminUsersTable.tsx:19]: @Zjjc123 this isn't too critical I guess you could move it to an issue

**Comment by daniel-sudz** [comment]
@Zjjc123 alright, too much to review but overall should be good. Let's test the preview to make sure nothing is too broken

**Comment by daniel-sudz** [comment]
@Zjjc123 tested manually on admin, looks good -- only thing functionality needed is adding those redirects on profile/table when you refresh but like we can add that in later

**Pull request merged**